src/bundle.ts

```diff
@@ -0,0 +1,87 @@
+import * as path from "path";
+import { SourceNode } from "source-map";
+import * as ts from "typescript";
+import { couldNotFindBundleEntryPoint } from "./diagnostics";
+import { resolveFromRootDir } from "./resolve";
+import { EmitHost, TranspiledFile } from "./Transpile";
+import { formatPathToLuaPath, trimExtension } from "./utils";
+
+const formatPath = (path: string) => formatPathToLuaPath(trimExtension(path));
+
+export function bundleTranspiledFiles(
+    bundleFile: string,
+    entryModule: string,
+    transpiledFiles: TranspiledFile[],
+    program: ts.Program,
+    emitHost: EmitHost
+): [ts.Diagnostic[], TranspiledFile] {
+    const diagnostics: ts.Diagnostic[] = [];
+
+    const resolvedEntryModule = resolveFromRootDir(program, entryModule);
```

Comment:
How about
```suggestion
    const entryModulePath = resolveFromRootDir(program, entryModule);
```

Reply:
Also, all paths specified in options should be resolved from project directory, not rootDir